pkg/zonemgr/zone_file.go

```diff
@@ -2,8 +2,9 @@ package zonemgr
 
 import (
 	"fmt"
-	v1 "kubevirt.io/api/core/v1"
 	"strconv"
+
+	v1 "kubevirt.io/api/core/v1"
 )
 
 const (
```

Comment:
Does it mean that the zone file will be refreshed only once in a hour?

Reply:
zone file is refreshed 45seconds according the reload in the auto plugin

Reply:
refresh = "3600"    // 1 hour (seconds) - how long a nameserver should wait prior to checking for a Serial Number increase within the primary zone file
	retry   = "3600"    // 1 hour (seconds) - how long a nameserver should wait prior to retrying to update a zone after a failed attempt.

Reply:
ic, 
but actually the auto plugin setting wins and it is 45 seconds

Reply:
@dteplits can we change it here to 45s as well? To be consistent.

Reply:
but it is not the same thing i believe
i won't change it
i think the auto plug does its magic enforcing the update
if there is no other update the internal one (maybe) try as well, if there is this kind of logic
it feels wrong to change it here

Reply:
ok, so let's verify what this "refresh" affects.

Reply:
I believe it is used when there is a chain of hierarchy
not on our case
see here "refresh" please
https://www.ionos.com/digitalguide/hosting/technical-matters/soa-record/

Reply:
btw we might want to see / understand what happens if that refresh time is reached without any timeout of us
for that we can make it lets say 3 minutes and see that there isn't some fatal